src/format.py

```diff
@@ -108,6 +110,27 @@ def getScreenIndex(self):
     def getPath(self):
         return self.path
 
+    def getSizeInBytes(self):
+        size = os.stat(self.path).st_size
+        return 'size: ' + str(size) + ' bytes'
+
+    def getLengthInLines(self):
+        num_lines = sum(1 for line in open(self.path))
+        return 'length: ' + str(num_lines) + ' lines'
```

Comment:
is it more efficient to do `cat $file | wc -l` via bash? I guess both have to open the whole file, just one is in python